client/src/styles/_bootstrap-variables.scss

```diff
@@ -711,7 +711,7 @@ $popover-body-padding-x:            $popover-header-padding-x; // SS custom
 // Modals
 
 // Padding applied to the modal body
-$modal-inner-padding: 20px;
+$modal-inner-padding: 1.25rem;
```

Comment:
How come this has been changed?

Reply:
Tests were failing cause it was used with another `rem` variable which isn't allowed anymore.